ansible/playbooks/roles/node/tasks/main.yml

```diff
@@ -6,6 +6,7 @@
     - '{{ ansible_distribution }}-{{ ansible_distribution_major_version }}.yml'
     - '{{ ansible_distribution }}.yml'
     - '{{ ansible_os_family }}.yml'
+    - '{{ cloud_platform }}-{{ ansible_distribution }}.yml'
```

Comment:
good one, can we change our aws/gcp yml files using this format?

Reply:
sure. but need collect aws/gcp's ansible_distribution content in advance